commands/list_test.go

```diff
@@ -66,3 +66,57 @@ func TestListAll(t *testing.T) {
 		"false", "https://example.org/p1/",
 	})
 }
+
+func TestBuildTimeFlag(t *testing.T) {
```

Comment:
I would prefer if you moved this to a "site test" in `hugo_test.go` and use the newly added `newTestHugoCmdBuilder` builder.